charts/flame-hub/README.md

```
@@ -82,11 +79,34 @@ client-secret:
 Create a secret:
 ```bash
 kubectl create secret generic flame-hub-auth \
-  --from-literal=postgresql-password=verysecurepassword \
   --from-literal=harbor-admin-password=anothersecurepassword \
   # ...
 ```
 
+> **Note:** The chart-managed Secrets carry `helm.sh/resource-policy: keep, so they are not removed upon a `helm uninstall`. This is because PVCs are also retained so they share the same lifecyle.
```

markdownlint-cli2 (0.23.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)